src/server.c

```diff
@@ -625,6 +625,17 @@ const char *strChildType(int type) {
     /* clang-format on */
 }
 
+
+const char *strRDBLoadType(int type) {
+    /* clang-format off */
```

Comment:
can we please avoid these clang-format directives and make sure the code is aligned with the clang format?

Reply:
removed